components/brave_new_tab_ui/components/default/braveToday/customize/Popular.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import styled from 'styled-components'
+import Button from '$web-components/button'
+import { useBraveNews } from './Context'
+import DiscoverSection from './DiscoverSection'
+import FeedCard from './FeedCard'
+import { getLocale } from '$web-common/locale'
+
+const LoadMoreButtonContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  grid-column: 2;
+`
+
+const DEFAULT_SUGGESTIONS_COUNT = 3
+
+export default function Suggestions () {
+  const { filteredPublisherIds, publishers } = useBraveNews()
+  const [showAll, setShowAll] = React.useState(false)
+  const popularPublisherIds = React.useMemo(() => filteredPublisherIds.map(id => publishers[id])
+    .filter(p => p.rank !== 0) // Rank of zero is unranked
```

Comment:
I think we probably want to include these, but leave them at the end. If we alphabetize first then at least they'll have _some_ order

Reply:
Done :smile: